test/helper/helper.go

```diff
@@ -0,0 +1,95 @@
+package helper
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	openapiclient "github.com/equinix-labs/metal-go/metal/v1"
+)
+
+func TestClient() *openapiclient.APIClient {
+	configuration := openapiclient.NewConfiguration()
+	configuration.AddDefaultHeader("X-Auth-Token", os.Getenv("METAL_AUTH_TOKEN"))
+	apiClient := openapiclient.NewAPIClient(configuration)
+	return apiClient
+}
+
+func Create_test_project(name string) string {
+	TestApiClient := TestClient()
+
+	projectCreateFromRootInput := *openapiclient.NewProjectCreateFromRootInput(name) // ProjectCreateFromRootInput | Project to create
+	include := []string{"Inner_example"}
+	exclude := []string{"Inner_example"}
+
+	projectResp, r, err := TestApiClient.ProjectsApi.CreateProject(context.Background()).ProjectCreateFromRootInput(projectCreateFromRootInput).Include(include).Exclude(exclude).Execute()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error when calling `ProjectsApi.CreateProject``: %v\n", err)
+		fmt.Fprintf(os.Stderr, "Full HTTP response: %v\n", r)
+		return ""
```

Comment:
All of these functions should return errors when they happen; if setup or teardown fails we want that to be surfaced as a test failure.